crates/autopilot/src/infra/persistence/mod.rs

```diff
@@ -77,25 +87,31 @@ impl Persistence {
             .map_err(Error::DbError)
     }
 
-    /// Inserts the given events with the current timestamp into the DB.
-    /// If this function encounters an error it will only be printed. More
-    /// elaborate error handling is not necessary because this is just
-    /// debugging information.
-    pub fn store_order_events(&self, events: Vec<(domain::OrderUid, boundary::OrderEventLabel)>) {
+    /// Inserts an order event for each order uid in the given set.
+    /// Unique order uids are required to avoid inserting events with the same
+    /// label within the same order_uid. If this function encounters an error it
+    /// will only be printed. More elaborate error handling is not necessary
+    /// because this is just debugging information.
+    pub fn store_non_subsequent_label_order_events(
```

Comment:
nit: Not sure we need to leak to "non subsequent storage" implementation detail to the domain (as it doesn't really care how we store the data). I'd just call it like before.

Reply:
Renamed